### ISSUE TITLE
React component methods may only be bound to the component instance.

### ISSUE THREAD

**Issue by brandondurham** [hongymagic/react-number-input#8]
Getting the following error:

```
bind(): React component methods may only be bound to the component instance. See exports
```

This is how I'm using the component:

```javascript
<NumberInput
	className = { styles.NumberInput }
	max = { 999 }
	min = { 1 }
	onChange = { this.onAppCountChange }
	type = 'number'
	value = { this.state.subscriptions.renewalCount }
/>
```

… and the `onChange` function:

```javascript
onAppCountChange(event) {
	var value = event.target.value
	var currentAppCount = this.state.subscriptions.appAccountTitles.length
	SubscriptionActions.updateAppRenewalCount(value)
	this.setState({
		errorMessage: value < currentAppCount ? 'You currently have ' + currentAppCount + ' active ' + _('app').pluralize(currentAppCount) + '. Tktk remove some' : null
	})
}
```

Any idea where this error is coming from?

**Comment by hongymagic** [comment]
Which version of react and react-number-input are you using? This _warning_ is most commonly found when using `bind` incorrectly (by author or component writers). Only `bind` this component has is to its own component methods, so I'm not sure where that would be coming from.

Is the `this.onAppCountChange` instance method bound to something elsewhere in the code?

**Comment by hongymagic** [comment]
If you were using `react-number-input@0.12.x` this issue should be fixed in `react-number-input@0.12.8`.
